docker/worker/run.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#set -e
+
+if [ -z $1 ]; then
```

Comment:
```suggestion
if [[ -z $1 ]]; then
```
since this is using bash anyways. Otherwise `$1` should be quoted.